packages/alfa-dom/test/jsx.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import { test } from "@siteimprove/alfa-test";
+
+test("Box attribute gets turned into box property", (t) => {
+  const element = <div box={{ x: 12, y: 7, width: 3, height: 9 }}></div>;
+
+  t.deepEqual(element.box.getUnsafe().toJSON(), {
+    type: "rectangle",
+    x: 12,
+    y: 7,
+    width: 3,
+    height: 9,
+  });
+});
+
+test("Element without box attribute has box property equal to `None`", (t) => {
+  const element = <div></div>;
+
+  t(element.box.isNone);
```

Comment:
```suggestion
  t.deepEqual(element.box.isNone(), true);
```
I think the current test is just making a truthy test on the function, which is passing because functions are not falsy.

Reply:
Good catch 🙈